episode-6.msrt

```diff
@@ -254,7 +252,7 @@
 51
 00:03:43,310 --> 00:03:47,310
 [ru] Зелье Зомбификации... глупости какие!
-[en] A potion of Zombification... what nonsense!
+[en] A potion of Zombification... that's nothing!
```

Comment:
@scribblemaniac is right, though if the "a" is very important to you, you could use a compound noun such as "what a nonsensical idea | what a nonsensical notion | what a nonsensical potion"